sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,33 @@
+
+.map-control {
+  --control-border-color: var(--bloom-color-blue-900);
+  --control-border-radius: var(--doorway-border-100);
+
+  z-index: 1;
+  flex: 1;
+  flex-direction: column;
+  position: absolute;
+  top: var(--bloom-s8);
```

Comment:
In the mocks, this is shown at the bottom left of the map, so shouldn't this be `bottom`, instead of `top`?

Reply:
There is a comment from Liann that is now resolved where she said it would look better at the top to avoid the Google logo. Having it at the top also works better for the mobile hybrid view since the listings list or the swipeable area are at the bottom of the screen and could cover the zoom controls.

Reply:
Thanks! That comment must have been resolved so I didn't see it.